apps/ledger-live-desktop/src/renderer/modals/Send/steps/Confirmation/NodeError/HelpSection.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Flex, Text, Icons } from "@ledgerhq/react-ui";
+import { useTranslation } from "react-i18next";
+import styled from "styled-components";
+
+type Props = {
+  onGetHelp: () => void;
+};
+
+const InteractFlex = styled(Flex)`
+  &:hover {
+    cursor: pointer;
+  }
+  padding: 8px;
+  background-color: ${({ theme }) => theme.colors.palette.opacityDefault.c05};
+  border-radius: 8px;
+  column-gap: 8px;
+`;
```

Comment:
The buttons are also visibly too big.
<img width="362" alt="image" src="https://github.com/user-attachments/assets/d9b51f9c-bb60-4cb3-9483-3440562d383a">

On Figma the font size should be 13px (so basically it should matches the other text font size):
<img width="441" alt="image" src="https://github.com/user-attachments/assets/b53878fe-e8cd-49cd-98a4-1494bfd41299">

As a result the size of the buttons and icons should be reduced too.

Reply:
Done ✅